app/lib/evss/claims_service.rb

```diff
@@ -0,0 +1,33 @@
+module EVSS
```

Comment:
my vote is `lib/services` prescriptions currently has a va_rx gem that does the client and middleware stuff, and for us it made a lot of sense to encapsulate as a gem at the time, but will likely move that into `lib/services` I think this will be clean and sensible alternative to having a bunch of gems... that being said, I also think there will probably be common concerns between services, but at some point we can add a `lib/services/concerns` and mix those in to keep DRY